server/target/target.go

```diff
@@ -893,13 +907,13 @@ func GetTargetStats(ctx context.Context, env environment.Env, req *trpb.GetTarge
 				duration_usec,
 				last_value(status) OVER win AS last_status
 			FROM "TestTargetStatuses"
-			WHERE (%s AND (status BETWEEN 1 AND 4))
+			WHERE (` + innerWhereClause + ` AND (status BETWEEN 1 AND 4))
 			WINDOW win AS (
 				PARTITION BY label
 				ORDER BY invocation_start_time_usec ASC
 				ROWS BETWEEN 1 PRECEDING AND 1 FOLLOWING))
 		WHERE (first_status BETWEEN 1 AND 2) AND (last_status BETWEEN 1 AND 2) AND status IN (3, 4) GROUP BY label) lf
-	ON lf.label=stats.label ORDER BY total_flakes DESC LIMIT 500`, innerWhereClause, innerWhereClause)
+	ON lf.label=stats.label ORDER BY total_flakes DESC LIMIT 500`
 
 	rq := env.GetOLAPDBHandle().NewQuery(ctx, "get_target_stats").Raw(qStr, qArgs...)
 	type qRow struct {
```

Comment:
theres one more place to update below (`getTargetFlakeSamples`)

Reply:
Good catch, done